app/src/main/java/chat/rocket/android/contacts/ContactsFragment.kt

```diff
@@ -113,9 +127,8 @@ class ContactsFragment : Fragment() {
 
         val searchItem = menu.findItem(R.id.action_search)
         searchView = searchItem?.actionView as? SearchView
-        searchView?.setIconifiedByDefault(false)
-        searchView?.maxWidth = Integer.MAX_VALUE
-        searchView?.onQueryTextListener { queryContacts(it) }
+        if (Constants.WIDECHAT) {
+        setupWidechatSearchView()}
```

Comment:
In the non WIDECHAT case it looks like you have broken the search by not including the onQueryTextListener() inside expandListener()